docs/vcf_files_preprocessor.md

```diff
@@ -0,0 +1,117 @@
+# VCF files preprocessor
+
+The VCF files preprocessor is used for validating the datasets such that the
+inconsistencies can be easily identified. It can be used as an individual
+validator to check the validity of the information inside the VCF files, or as a
+helper tool for [VCF to BigQuery pipeline](/README.md/#vcf_to_bq). The VCF to
+BigQuery loading process may fail for ill-defined datasets, which can be
+resolved by applying
+[conflicts resolving strategies](https://docs.google.com/document/d/1JusaPllbpXynNziNzdyOdSRTM8i7UXbNvtkbrWNMq3Q).
```

Comment:
For now, I think linking to `multiple_files.md#field-compatibility` is good enough. We should update that file to have the new logic (@nmousavi , FYI). I think linking to that design doc is too detailed.